### PR TITLE
Update doc to support installation on IBM Power.[skip ci]

### PR DESCRIPTION
**Pull request by sarveshtamba**
**What type of PR is this?**
/kind documentation
[skip ci]

**What does does this PR do / why we need it**:
Since the odo alpha-2.0 test suites has passed for all tests on Power, The alpha-2.0 installation should be added to doc for users of odo on IBM Power. Similar to PR https://github.com/openshift/odo/pull/3914

**Which issue(s) this PR fixes**:

Fixes #?

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [ ] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by sarveshtamba** [comment]
@amitkrout can this one be approved and merged?

**Comment by amitkrout** [comment]
> @amitkrout can this one be approved and merged?

/lgtm

File: docs/public/installing-odo.adoc [hunk 1]
```diff
@@ -24,6 +24,15 @@ v2.0.0-alpha of odo uses **devfile** as its default deployment mechanism.
 # chmod +x /usr/local/bin/odo
 ----
 
+== Installing odo on Linux on IBM Power
+
+=== Binary installation
+
+----
+# curl -L https://github.com/openshift/odo/releases/download/v2.0.0-alpha-2/odo-linux-ppc64le -o /usr/local/bin/odo
```
Comment (amitkrout) [inline docs/public/installing-odo.adoc:32]: @sarveshtamba We have GA bits https://github.com/openshift/odo/releases/tag/v2.0.0. So can you please update the references in the doc and commit it into ```master``` branch not ```openshift:alpha```

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: docs/public/installing-odo.adoc (hunk 1 above)
Comment (sarveshtamba) [reply docs/public/installing-odo.adoc:32]: PR for master branch has already been merged here https://github.com/openshift/odo/pull/4049
As per discussion closing this PR # 4055 as alpha branch is no longer used.

**Comment by sarveshtamba** [comment]
PR for master branch has already been merged here #4049
As per discussion closing this PR #4055 as alpha branch is no longer used.

**Pull request closed without merging**